docker-compose.yml

```diff
@@ -105,9 +106,10 @@ services:
       - net
     volumes:
       - ./configs/backend/:/etc/strelka/:ro
-    container_name: sublime_strelka_backend_1
```

Comment:
What will our container names be now? https://github.com/sublime-security/sublime-platform/pull/70 I think we'd prefer to ensure they're prefixed, but it's not a huge deal for these ones.

Reply:
Good Q. Just goes from `_X` to `-X`
![image](https://github.com/sublime-security/sublime-platform/assets/31489089/2602e3be-5bdf-4725-8db6-099f3543e07f)

I'll see if I can change the prefix so it's more hostname friendly, but I believe that we're okay with the `-` for these ones since nothing needs to know the name?

Reply:
oh wait nevermind, `-` is better for hostnames than `_` anyway